setup.py

```diff
@@ -1,9 +1,35 @@
 #!/usr/bin/env python
+import os
 from setuptools import setup, find_packages
 
 with open("README.rst") as f:
     long_description = f.read()
 
+data_files = []
+bin_files = []
+cmdclass = {}
+
+bin_license = 'docs/License.html'
+if os.path.exists(bin_license):
+    data_files.append(('docs', [bin_license]))
+    bin_files.extend(['MediaInfo.dll', 'libmediainfo.*'])
```

Comment:
How does it work on Linux, the library and the license aren't present in the current folder, are they?